tests/Constraints/AdditionalPropertiesTest.php

```diff
@@ -0,0 +1,38 @@
+<?php
+/**
+ * This file is part of the Gruvo software.
```

Comment:
Would you be OK with me just updating the license to say "Copyright (c) 2016 Matthew Allan matthew.james.allan@gmail.com and the JSON Guard contributors" (as [described here](http://ben.balter.com/2015/06/03/copyright-notices-for-websites-and-open-source-projects/#the-copyright-holder) instead of adding the copyright header?